code/modules/vending/autodrobe.dm

```diff
@@ -215,7 +215,14 @@
 		/obj/item/clothing/under/dress/wedding_dress = 1,
 		/obj/item/clothing/under/suit/tuxedo = 1,
 		/obj/item/clothing/head/costume/weddingveil = 1,
+<<<<<<< HEAD
 		/obj/item/storage/belt/cummerbund = 1,
+=======
+		/obj/item/storage/belt/fannypack/cummerbund = 1,
+		/obj/item/clothing/suit/costume/drfreeze_coat = 1,
+		/obj/item/clothing/under/costume/drfreeze = 1,
+		/obj/item/clothing/head/costume/drfreezehat = 1,
+>>>>>>> 84182bc05cf (Fixes a oversight with cummerbund. closes #72357 (#72373))
```

Comment:
```suggestion
		/obj/item/storage/belt/fannypack/cummerbund = 1,
```
### **!!Missed mirror of https://github.com/tgstation/tgstation/pull/72103!!**
PR can be merged without the items in the autodrobe for now, though

Reply:
https://github.com/Skyrat-SS13/Skyrat-tg/pull/18638 first just in case